.settings/.settings/org.eclipse.m2e.core.prefs

```diff
@@ -0,0 +1,4 @@
+activeProfiles=
+eclipse.preferences.version=1
+resolveWorkspaceProjects=true
+version=1
```

Comment:
I think `org.eclipse.m2e.core.prefs` can be removed, it's no longer a Maven project.